src/System.Management.Automation/engine/parser/ast.cs

```diff
@@ -753,14 +753,6 @@ public class ScriptRequirements
         /// </summary>
         public ReadOnlyCollection<ModuleSpecification> RequiredModules { get; internal set; }
 
-        /// <summary>
-        /// The snapins this script requires, specified like:
-        ///     <code>#requires -PSSnapin Snapin</code>
-        ///     <code>#requires -PSSnapin Snapin -Version 2</code>
-        /// If no snapins are required, this property is an empty collection.
-        /// </summary>
-        public ReadOnlyCollection<PSSnapInSpecification> RequiresPSSnapIns { get; internal set; }
```

Comment:
This is a public API, so removing it is a breaking change. Would it be acceptable?

Reply:
I don't find anything on GitHub using `RequiresPSSnapIns` except for PowerShell repo itself:[RequiresPSSnapIns - grep.app](https://grep.app/search?q=RequiresPSSnapIns)

So I think it's OK to make this breaking change